Tests/ScipioKitTests/PartialCacheTests.swift

```diff
@@ -0,0 +1,57 @@
+import Foundation
+import Testing
+@testable import ScipioKit
+import Logging
+
+private let fixturePath = URL(fileURLWithPath: #filePath)
+    .deletingLastPathComponent()
+    .appendingPathComponent("Resources")
+    .appendingPathComponent("Fixtures")
+
+struct PartialCacheTests {
+    private let fileManager: FileManager = .default
+    private let tempDir: URL
+    private let frameworkOutputDir: URL
+
+    init() throws {
+        LoggingSystem.bootstrap { _ in SwiftLogNoOpLogHandler() }
```

Comment:
I tried to run the tests for this PR, it seems that we should remove this line to pass the tests.

Maybe this should be called once during the entire test execution.

Reply:
Removed https://github.com/giginet/Scipio/pull/184/commits/34f7efcc8c98190afe8d904cdaac59dfcb234177